src/main/java/com/pragbits/bitbucketserver/components/PullRequestActivityListener.java

```diff
@@ -55,10 +56,10 @@ public void pullRequestEvent(PullRequestActivityEvent event) {
 
 	public void notifySlackUsers(PullRequestActivityEvent event) {
 		Repository repository = event.getPullRequest().getToRef().getRepository();
-		
+
+		Set<ApplicationUser> users = event.getPullRequest().getWatchers().stream().map(w -> w.getUser()).collect(Collectors.toSet());
```

Comment:
I like the formatting style where you put each stream action on a separate line, eg:

```
		Set<ApplicationUser> users = event.getPullRequest().getWatchers().stream()
                                  .map(w -> w.getUser())
                                  .collect(Collectors.toSet());
```

I tried to find other stream usages in the code base but couldn't, and in this case with only a map and collect it's not a huge difference anyway. Pick what you think is best.